keyboards/xd60/keymaps/styr/keymap.c

```diff
@@ -0,0 +1,62 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```

Comment:
Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.

Reply:
should `#include QMK_KEYBOARD_H` be removed as well ?

Reply:
No.  That should be left.  

Specifically, `QMK_KEYBOARD_H` is the keyboard's h file.  This file should include `quantum.h`, which includes many other files, including `action_layer.h`.  

So, in most cases, you don't need anything but `QMK_KEYBOARD_H` included.

Reply:
Perfect! All removed.